numpy/core/src/multiarray/compiled_base.c

```diff
@@ -1504,6 +1504,17 @@ arr_add_docstring(PyObject *NPY_UNUSED(dummy), PyObject *args)
 #include <emmintrin.h>
 #endif
 
+#if defined __ARM_NEON || defined __ARM_NEON__
+    #include <arm_neon.h>
+    int sign_mask(uint64x2_t input)
+    {
```

Comment:
Define it as an inline function, via `static NPY_INLINE`.
change the parm type to `uint8x16_t`, remove the cast and add 'u8' as a suffix to increase the readability.

Reply:
modified